chromium_src/crypto/obsolete/md5.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CRYPTO_OBSOLETE_MD5_H_
+#define BRAVE_CHROMIUM_SRC_CRYPTO_OBSOLETE_MD5_H_
+
+#define KnownAnswer                                                    \
```

Comment:
what are we doing here and why? Remember to include explanations in comments for overrides

Reply:
Added a comment explaining this here https://github.com/brave/brave-core/pull/32111/commits/ad2883a5bff439f6b5d6f8a948b1b10631f98713.

In a nutshell, though, `Md5` now requires you to create a helper function for MD5 usage and make it a friend of the `Md5` class.